lib/src/router/router.dart

```diff
@@ -101,7 +100,14 @@ class AppRouter {
                 builder: (_, state) {
                   final id =
                       state.pathParameters[AppRouteNames.story.paramName]!;
-                  return StoryView(id: id);
+                  final extra = state.extra! as List<dynamic>;
```

Comment:
Change null check
Btw, you should create extra models for screens that need to pass params like this instead of using `List<dynamic>`
For example in this case:
```
class StoryViewExtra {
final DataType paramA;
final DataType paramB;
...
}
```